scripts/wave-lane-watch.ps1

```
@@ -0,0 +1,315 @@
+  $lanes = @(Get-Lanes -ExplicitBranches $Branches)
+  $rows = @()
+  $i = 1
+  foreach ($lane in $lanes) {
+    $rows += Measure-Lane -branch $lane.branch -path $lane.path -laneIndex $i
+    $i++
+  }
+  return , $rows
+}
+
+# Frisk origin FOR maalingen (best-effort; hang ikke paa netvaerk).
+try { & git -C $repoRoot fetch --prune origin *> $null } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)